apps/librelingo_fakes/librelingo_fakes/fakes.py

```diff
@@ -113,6 +113,24 @@
     in_target_language=["apfel"], in_source_language=["apple"], pictures=["1", "2", "3"]
 )
 
+pairs1 = [Word(
```

Comment:
I think this is the only "major" issue see in this PR. :thinking:  why are the variables `pairs1` and `pairs2` even needed? I mean, they are just lists of words. But skills are already lists of words. In the other challenge types fake skills are used for this purpose I think. Which is how it should be done here as well.

Reply:
I am having a bit of trouble trying to use fake skills. In challenges.py, I can no longer call get_pairing_challenge under _get_phrase_challenges or _get_word_challenges as I get this error "ERROR test_challenges_get_phrase_challenges.py - AttributeError: 'Phrase' object has no attribute 'words'". I am unable to find a function that will pass in a skill. Just wondering if you have any tips?

Reply:
You should probably use `skillWith3Words` from fakes.py in order to achieve what you want. I think everything should work then :thinking:

Reply:
Yes, that is what I am currently trying to use, however I am having a bit of trouble on how to call the pairing function in challenges.py now. 

![Screenshot from 2022-04-17 12-29-55](https://user-images.githubusercontent.com/55596950/163723781-ca7a9f61-38d9-4f62-8f50-515a54003e3d.png)